modules/multi-arch-installing-using-cli.adoc

```diff
@@ -0,0 +1,120 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multiarch-tuning-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-installing-using-cli_{context}"]
+
+= Installing the Multiarch Tuning Operator by using the CLI
+
+You can install the Multiarch Tuning Operator by using the {oc-first}.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have installed the {oc-first}.
+
+.Procedure
+
+. Log in to the {oc-first}.
+
+. Create a new project named `openshift-multiarch-tuning-operator` by running the following command:
++
+[source,terminal]
+----
+$ oc new-project openshift-multiarch-tuning-operator
+----
+
+. Create an `OperatorGroup` object:
+
+.. Create a YAML file with the configuration for creating an `OperatorGroup` object.
++
+.Example YAML configuration for creating an `OperatorGroup` object:
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-multiarch-tuning-operator
+  namespace: openshift-multiarch-tuning-operator
+spec: {}
+----
+
+.. Create the `OperatorGroup` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <file_name>
```

Comment:
```suggestion
$ oc create -f <file_name> <1>
```
add footnote, something like
<1> Replace file_name with the name of the OperatorGroup object...

Reply:
> the name of the OperatorGroup object...

it should be the path to the file containing the object.

Reply:
Added the following info:

<1> Replace `<file_name>` with the name of the YAML file that contains the `OperatorGroup` object configuration.